### PR TITLE
Changes made in mode and purpose

### PR DESCRIPTION
**Pull request by deepalics0044**

### CODE REVIEW

File: www/json/connectionConfig.physical_device2localhost.json.sample [hunk 1]
```diff
@@ -2,13 +2,13 @@
     "connectUrl": "http://<your laptop ip>:8080",
     "android": {
         "auth": {
-            "method": "prompted-auth",
+            "method": "dummy-dev",
```
Comment (shankari) [inline www/json/connectionConfig.physical_device2localhost.json.sample]: Intentional?

File: www/json/connectionConfig.physical_device2localhost.json.sample [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
             "clientID": "ignored"
         }
     },
     "ios": {
         "auth": {
-            "method": "prompted-auth",
+            "method": "dummy-dev",
```
Comment (shankari) [inline www/json/connectionConfig.physical_device2localhost.json.sample]: Intentional?

**Comment by shankari** [comment]
@deepalics0044 is there going to be an update to this PR?

**Comment by deepalics0044** [comment]
Yes ma'am

**Comment by shankari** [comment]
@deepalics0044 you still have changes in 4 files, including the deletion of `www/json/connectionConfig.openid.json.sample` and ` www/json/connectionConfig.token_list.json.sample`. Those are primarily sample files, so theoretically I can merge them, but I am concerned that having unrelated changes will cause you to have a merge conflict the next time you pull from master.

Is there a reason you did not revert those changes either?

**Comment by deepalics0044** [comment]
Can you please merge the changes, we need it fast.

**Comment by shankari** [comment]
@deepalics0044 ok, but I would strongly recommend fixing the deletes so that you don't run into further problems down the road.

**Comment by shankari** [comment]
Deployed, should be live in ~ 10 mins

**Pull request merged**